cmake/configs/nuttx_px4fmu-v4_default.cmake

```diff
@@ -80,9 +80,7 @@ set(config_module_list
 	# Too high RAM usage due to static allocations
 	# modules/attitude_estimator_ekf
 	modules/attitude_estimator_q
-	modules/ekf_att_pos_estimator
-	modules/position_estimator_inav
-	modules/ekf2
```

Comment:
And neither ekf_att_pos_estimator - the fixed wing configs rely on it. Don't change things that you're not forced to change.